mk/Common.mk

```diff
@@ -353,6 +353,13 @@ include $(CFG_FLAGS_MAKE)
 endif
 endif
 
+ifeq ($(CV_CORE),cv32e40p)
```

Comment:
I really dislike this.  The remapping was just a bad idea from the get-go and this makes it an orphan. I've created #768 to discuss "unorphanning" the CV32E40P.

Reply:
Me too.  Supporting the proper UVM memory in E40P should be relatively easy, but will want to have a discussion on the core testbench + Verilator.  I can address 768.